doc/update_organizations.md

```diff
@@ -0,0 +1,53 @@
+# Updating Partner Organizations
+
+This guide explains how to update the partner organizations listed on the website. The data for these organizations is stored in a JSON file located in the `src/assets/data` directory.
+
+## Partner Organizations
+
+Partner organizations are listed in the `partner-organizations.json` file. To update the partner organizations, follow these steps:
+
+1. Open the `src/assets/data/partner-organizations.json` file.
+2. Add, remove, or update the partner organization entries as needed. Each entry should have the following structure:
+
+```jsonc
+{
+  "name": "Organization Name",
+  "img": "logo_filename.webp",
+  "width": 160,
+  "height": 72,
+  "featured": 5, // optional, from 1 to 5
+}
+```
+
+You can add a **"featured"** flag on an organization to display it in the homepage, in the order you chosen (1 to 5).
```

Comment:
just to be sure to understand : only features partners tagged with 1 to 5 are displayed in the homepage ? 
if i put 6, it will never been displayed ?

Reply:
Yes it's only the 5 first elements. Let me clarify this in the doc 👍